src/__tests__/CircleBadge.test.tsx

```diff
@@ -1,11 +1,9 @@
 import React from 'react'
 import {CircleBadge} from '..'
 import {CheckIcon} from '@primer/octicons-react'
-import {render, mount, behavesAsComponent, checkExports} from '../utils/testing'
+import {render, behavesAsComponent, checkExports} from '../utils/testing'
 import {render as HTMLRender} from '@testing-library/react'
-import {axe, toHaveNoViolations} from 'jest-axe'
-
-expect.extend(toHaveNoViolations)
+import {axe} from 'jest-axe'
```

Comment:
It is not a part of enzyme removal but what do you think using `checkStoriesForAxeViolations` from our testing utils instead of directly importing `jest-axe`? Totally cool, if you think it is not a part of this PR 👍🏼